app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonFragment.kt

```diff
@@ -18,7 +19,9 @@ import javax.inject.Inject
 /** Fragment that allows the user to resume a saved exploration. */
 class ResumeLessonFragment : InjectableFragment() {
   companion object {
-    private const val ARGUMENTS_KEY = "ResumeExplorationFragment.arguments"
+
+    /** Arguments key for [ResumeLessonFragment]. */
+    const val ARGUMENTS_KEY = "ResumeExplorationFragment.arguments"
```

Comment:
```suggestion
    const val RESUME_LESSON_FRAGMENT_ARGUMENTS_KEY = "ResumeLessonFragment.arguments"
```

There are multiple instances of `ARGUMENTS_KEY` in the codebase, and an incorrect import could lead to errors.